src/scss/next.scss

```diff
@@ -394,6 +394,7 @@ details summary {
   font-size: get-size('size-2');
   list-style: none;
   cursor: pointer;
+  padding-right: 28px;  // Prevent overlap with ::before element
```

Comment:
Can you try `get-space('size-1')` on this please?

Reply:
Should I then also use this on ::before height&width?